app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -219,6 +223,11 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         super.onBackPressed()
     }
 
+    override fun dispatchTouchEvent(ev: MotionEvent): Boolean {
+        FullscreenTransitionMeasurement.onHomeActivityDispatchTouchEvent(ev) // DO NOT MOVE ANYTHING ABOVE HERE.
```

Comment:
same question as above, is there a reason why?

Reply:
Same answer. :)